mimic/test/test_images.py

```diff
@@ -0,0 +1,113 @@
+"""
+Tests for :mod:`nova_api` and :mod:`nova_objects` for images.
+"""
```

Comment:
Why is this in a separate file. We could merge this with the existing test_server_images.py file as this seems redundant.

Also, as mentioned below, if the idea to validate IAD and non IAD regions, lets have separate test classes with the same file.